### ISSUE TITLE
No images showing until scrolled to the top of the gallery

### ISSUE THREAD

**Issue by monsterchops** [nanostudio-org/nanogallery2#236]
Currently none of the images show up at all until I scroll to the top of the gallery or past the section where the gallery resides.  The gallery is seen in full height, ... i.e. the image dimensions take up space but the images themselves are not showing.  I reduced the number of images to load to 4 to ensure the issue is not with the pure volume of images or bandwidth.  Admittedly in this case I do not have thumbnails and therefore am loading the same image into the anchors href & data-ngthumb attributes.  It seems like an effect or feature that is happening that I just need to remove but I can't figure out how.  So far I've tried setting thumbnailDisplayOutsideScreen to true and galleryDisplayTransition to null as some have suggested but nothing seems to effect it.  Any ideas on what setting I might be missing here?  Thank you.

**Comment by monsterchops** [comment]
Update: I meant galleryDisplayTransition to none didn't seem to effect it.  Note, I also have tried the markup method and the Javascript method both with the same results fyi.

**Comment by monsterchops** [comment]
This issue was ultimately related to the markup surrounding the gallery.  The design I came up with is a combination of old school image face plate meets responsive website on steroids so there is a lot of extra markup that you probably would not see in a standard responsive Bootstrap site. I stripped down the code until it worked and then built it back up again until I found the offending bit.  This is a really nice gallery! Thank you.
